Makefile

```diff
@@ -771,3 +771,11 @@ test-packages: packages packaging/rpm/centos-systemd/$(UPTODATE) packaging/deb/d
 
 docs: doc
 	cd docs && $(MAKE) docs
+
+.PHONY: generate-otlp
+generate-otlp:
+	cd pkg/distributor/otlp && go generate
+
+.PHONY: check-generated-otlp-code
```

Comment:
From the weekly: we have a general pattern to pick up changes: first delete the generated files `*_generated.go` and then re-generate and check if there's any change picked up by git. See the use of `./tools/find-diff-or-untracked.sh` in the Makefile. This handles files missing/added and if the generator itself fails.

Reply:
Thanks, done.